projects/observability/src/shared/graphql/request/handlers/entities/query/entity/entity-graphql-query-handler.service.ts

```diff
@@ -61,7 +61,7 @@ export class EntityGraphQlQueryHandlerService implements GraphQlQueryHandler<Gra
       // If querying for a single API by ID, then usually want to includeInactive
       includeInactive: request.includeInactive !== false,
       // If querying for a single API by ID, ignore all global filters on entities query
-      ignoreGlobalFilters: true
+      ignoreApiDiscoveryStateFilter: true
```

Comment:
I would change the entity query handler to not ever apply global query handlers, rather than try to pick and choose which should be ignore and which shouldn't.